### PR TITLE
Added GetLoadedTypes to DependencyScanner and changed usages of GetTypes() to use that.

### LINKED ISSUE #58: Fluxor does not work with LiveSharp.
Hi @mrpmorris,

I am using LiveSharp, a tool to rerender changes to the browser immediately. This tool sadly does not work with Fluxor at the moment. 

I've reported the problem to the maintainer of LiveSharp, but he can't find the solution on his end, and it seems the problems lies in a missing try/catch when resolving assemblies with Fluxor. Is this something you can solve on your end?

More information can be found in the following issue: [https://github.com/OYIon/LiveSharp/issues/78](https://github.com/OYIon/LiveSharp/issues/78)

### PR DESCRIPTION
**Pull request by niem94**
Fixes #58

### CODE REVIEW

File: Source/Fluxor/DependencyInjection/DependencyScanner.cs [hunk 1]
```diff
@@ -112,5 +112,16 @@ internal static class DependencyScanner
 				return store;
 			});
 		}
+		 public static IEnumerable<Type> GetLoadedTypes(Assembly assembly)
```
Comment (mrpmorris) [inline Source/Fluxor/DependencyInjection/DependencyScanner.cs:115]: You have a mix of tabs and spaces in here, could you use only tabs please?

Comment (niem94) [reply Source/Fluxor/DependencyInjection/DependencyScanner.cs]: Should be fixed now :-)

**Comment by mrpmorris** [comment]
I'm concerned about ignoring types we have problems with.

For example, what if the type that cannot be loaded is an Effect. If we simply exclude it and continue with the app then we are effectively excluding parts of the user's application. Triggers won't fire, reducers will be missing and so on, resulting in an inconsist (and unpredictable application).

This really is something that needs to be fixed in either LiveSharp or SignalR Client, whichever it is that is introducing the dependency on an assembly that cannot be loaded.

I appreciate you submitting this (thank you), and I was on the fence about the solution until this morning.

I'm sorry I didn't think of these concerns before agreeing to you creating the PR.

Thanks again for your contribution!

**Comment by mrpmorris** [comment]
Confirmed problem in LiveSharp
https://github.com/mrpmorris/Fluxor/issues/58

**Pull request closed without merging**